src/test/java/uk/gov/ida/dropwizard/logstash/LogstashConsoleAppenderAppRuleTest.java

```diff
@@ -49,11 +49,15 @@ public void testLoggingLogstashRequestLog() throws InterruptedException, IOExcep
         assertThat(accessEventStream.size()).isEqualTo(1);
         AccessEventFormat accessEvent = accessEventStream.get(0);
         assertThat(accessEvent.getMethod()).isEqualTo("GET");
-        assertThat(accessEvent.getContentLength()).isEqualTo("hello!".length());
-        assertThat(accessEvent.getRequestedUri()).isEqualTo("/");
-        assertThat(accessEvent.getProtocol()).isEqualTo("HTTP/1.1");
-        assertThat(accessEvent.getStatusCode()).isEqualTo(200);
+        assertThat(accessEvent.getBytesSent()).isEqualTo("hello!".length());
+        assertThat(accessEvent.getUrl()).isEqualTo("/?queryparam=test");
+        assertThat(accessEvent.getHttpVersion()).isEqualTo("1.1");
+        assertThat(accessEvent.getResponseCode()).isEqualTo(200);
+        assertThat(accessEvent.getRemoteIp()).isEqualTo("127.0.0.1");
         assertThat(accessEvent.getVersion()).isEqualTo(1);
+        // ballpark check that the unit is in the right order of magnitude
+        // this test should hopefully catch a value that's erroneously measured in seconds
+        assertThat(accessEvent.getElapsedTimeMillis()).isBetween(3,3000);
```

Comment:
I wanted to be a tiny bit stronger than that so we could be reasonably sure it wasnt microseconds or nanoseconds either